src/git-ship

```diff
@@ -6,8 +6,8 @@ function error_empty_commit {
   discard_open_changes
   undo_command
 
-  echo_error_header
-  echo_error "Aborting ship due to empty commit message"
+  echo
+  echo_red "Aborting ship due to empty commit message"
```

Comment:
Changing output is a separate change that should be reviewed separately. Also, I'm for keeping the semantic formatting ("error") instead of the hard-coded one ("red"). If we ever change the error color, its a one-liner with semantic descriptions, and a huge shotgun refactor with the hard-coded ones.